tests/launcher_tests.js

```diff
@@ -170,6 +170,17 @@ describe('Launcher', function(){
       assert.equal(launcher.commandLine(), '"node -e console.log(process.argv.slice(1).join(\' \')) hello"')
     })
 
+    it('returns commandLine with a single exe', function(done){
+      settings = {exe: ['node', 'npm'], args: function(){ return ['-e', 'console.log(1)'] }}
```

Comment:
Removing function makes `commandLine` return something along `"node -e console.log(1) http://localhost:7357/1838"`. I didn't want to try matching URL and id as well so I ended up having function here.